qiskit/providers/fake_provider/backends/almaden/fake_almaden.py

```diff
@@ -15,7 +15,7 @@
 """
 
 import os
-from qiskit.test.mock import fake_pulse_backend, fake_backend
+from qiskit.providers.fake_provider import fake_pulse_backend, fake_backend
```

Comment:
Not in the scope of this PR, but should this line be changed to 

```python
from qiskit.providers.fake_provider import FakeQasmBackend, FakePulseBackend, FakeBackendV2
```

Reply:
Thanks for the comment. I think i will leave this out of this PR since it's not that important. I will create an issue as first good issue for others to contribute.